src/index.js

```diff
@@ -92,6 +92,7 @@ class Loading extends React.Component {
 
 Loading.propTypes = {
     isLoading: React.PropTypes.bool,
+    extraStyle: React.PropTypes.object,
```

Comment:
The prop should be just called `style` instead of `extraStyle` so that it's more intuitive to use. eg:

```html
<Loading style={{ display: 'inline-block', color: '#123456', marginLeft: 30, top: 10 }} />
```